packages/inter-protocol/scripts/price-feed-core.js

```diff
@@ -57,7 +57,7 @@ export const defaultProposalBuilder = async (
         brandOutRef: brandOut && publishRef(brandOut),
         priceAggregatorRef: publishRef(
           install(
-            '@agoric/inter-protocol/src/price/fluxAggregator.js',
+            '@agoric/inter-protocol/src/price/fluxAggregator.contract.js',
```

Comment:
no this is new. I'm finding it common to have a contract have a singleton and it seems silly to have separate names for them. E.g. vaultFactory has vaultDirector, but it's just one more name to keep track of. We often talk about their behavior with one name. So I think it better to keep the same common name for both but qualify one as the contract.

Reply:
okay. What's the argument for fluxAggregator.contract.js rather than fluxAggregatorContract.js?  I think the latter is more in keeping with our standard style. It seems odd to call some contracts out with .contract of we're not doing that in general.

I guessed that this was the start at a plan to rename all contracts, which sounds somewhat plausible, but I hadn't heard the arguments either way.

Reply:
> What's the argument for fluxAggregator.contract.js rather than fluxAggregatorContract.js? 

A contract file is not just any JS file because it has to have an export of `start` or `prepare`. I expect to also eventually autogenerate documentation about contracts from the contract file. I'd like to be able to autogenerate a report of contracts within a package.

The above could be done by looking for a `start` or `prepare` export, but that doesn't necessarily mean it's a contract. There should be some way to declare that some code is a contract that can be read cheaply. (e.g. no type analysis to see whether it implements the API)